server/templates.md

```diff
@@ -100,17 +74,16 @@ var data = require('./data')
 
 var app = express()
 
-app.engine('hbs', hbs())
+app.engine('hbs', hbs(
+  extname: 'hbs',
+  defaultLayout: 'main'
+))
 app.set('view engine', 'hbs')
 app.set('views', path.join(__dirname, 'views'))
```

Comment:
It's probably worth noting that you don't actually have to set the `views` directory: it's the default. Would it be worth leaving it out to keep the examples simple?